### ISSUE TITLE
Too many default extensions

### ISSUE THREAD

**Issue by W-Y-R** [schizofox/schizofox#84]
I acknowledge that extensions can enhance security from a user perspective, but they also make the browser’s fingerprint more unique. 
>Keep in mind that any "super ultra privacy friendly firefox config" will make you stick out and won't protect you from fingerprinting; sadly there is no escape from that. If you are really looking for security, look into the Tor browser.

This appears to be inconsistent with the overall purpose of Schizofox. Its default settings do not safeguard the user from fingerprinting, but instead compromise his privacy. For that matter, I think that it should come by default only with Dark Reader and uBlock Origin (which are the most common).

Interesting module otherwise, and I'm looking forward to what it will become in the future.

**Comment by NotAShelf** [comment]
I am open to culling default extensions to a minimum, as we've received criticism regarding the default bath of extensions previously. It should be noted, however, that it's entirely possible to disable the default extensions by using `lib.mkForce` on the `defaultExtensions` setting.

> I acknowledge that extensions can enhance security from a user perspective, but they also make the browser’s fingerprint more unique.

If we were to try and avoid fingerprinting, we'd have to be running a *stock* Firefox config (or even chrome) to stand out less. There is no escaping fingerprinting - anything that we add in hopes of security, will contribute to fingerprinting. See #76 on what is actually leaked.

> This appears to be inconsistent with the overall purpose of Schizofox. Its default settings do not safeguard the user from fingerprinting, but instead compromise his privacy. For that matter, I think that it should come by default only with Dark Reader and uBlock Origin (which are the most common).

As aforementioned, there is no escaping fingerprinting while also trying to provide a secure browser. If we focus on security, we'll be contributing to fingerprinting and vice versa. Are you suggesting that instead of providing defaults which "compromise privacy", we simply should ship a Firefox configuration with two extensions preinstalled?

**Comment by W-Y-R** [comment]
>It should be noted, however, that it's entirely possible to disable the default extensions by using lib.mkForce on the defaultExtensions setting.

The goal should be to provide a solid and secure base for users to build upon, not to impose unnecessary or unwanted features on them. 

>If we were to try and avoid fingerprinting, we'd have to be running a stock Firefox config (or even chrome) to stand out less. 

Yes, it's just as you said. The best way to avoid it is to blend in with the majority of users, which means that using a hardened stock Firefox would be a much better idea.

>Are you suggesting that instead of providing defaults which "compromise privacy", we simply should ship a Firefox configuration with two extensions preinstalled?

Upon further reflection, I believe that it should only include uBlock (it is also the default extension for Tor on Tails, which is a good indication of its reliability). 

>We should take another look at arkenfox's user.js, revisit our old policies and refactor where necessary.

Privacy and security wise, I recommend using the narsil user.js profile. I don't think there's any profile more "schizophrenic" than this, so it's the best for the scope of this project.

In summary, this is my suggested course of action for the project:
- JavaScript should be disabled by default
- Comes only with uBlock Origin
- The narsil user.js is the default profile
- The default search engine should be searx.be, instead of brave (business, not open source) (also searx.be is much more customizable)

**Comment by jh-devv** [comment]
+1

**Comment by mateodd25** [comment]
I think, user-js refers to https://github.com/pyllyukko/user.js

**Comment by jh-devv** [comment]
> I think user.js refers to https://github.com/pyllyukko/user.js

Probably not, since https://github.com/schizofox/schizofox/issues/76#issuecomment-1873099423

**Comment by NotAShelf** [comment]
I've reduced default extensions to just uBlock Origin similar to what Librewolf is doing. Testing would be appreciated.
